### DISCUSSION TITLE
Integration of RT feed behind API-key

### DISCUSSION THREAD

**Discussion by eMerzh** [transitland/transitland#423] [Q&A] [answered]
Hello 👋 

I see that my local transit company is referenced as feed provider here https://www.transit.land/feeds/f-u0g-tec
but it misses the RT part.

The RT seems behind an API KEY (i have one), 
i was wondering if it could be integrated in transitland ?
I can probably get a new key for transitland folks (or connect the two), but i'm not sure i'll be able to convert the whole thing to publicly key-free CC0 licence ...

**Comment by drewda** [comment, marked as answer]
There are two separate aspects to adding RT feeds with authentication:

1) adding a listing for the RT feed URL to the [Transitland Atlas feed registry](https://github.com/transitland/transitland-atlas). This listing includes URL and information about API auth, but it does not include an actual API key

2) the Interline team maintains a private list of API keys used by the www.transit.land stack. This is mapped against the Onestop IDs in Transitland Atlas. These API keys are not shared publicly.

You're welcome to make email introductions between agencies and the Interline team by emailing hello@transit.land, or you can share sign-up URLs and we'll sign up ourselves. Thank you.

**Reply by drewda** [reply]
An update: We just received documentation as well as a key to be able to add this feed to Transitland: https://github.com/transitland/transitland-atlas/pull/1263

@eMerzh Thank you again for these pointers tot he appropriate contacts.
